core/backend/src/BackendHubAccess.ts

```diff
@@ -232,6 +232,7 @@ export interface BackendHubAccess {
   /**
    * acquire one or more locks. Throws if unsuccessful. If *any* lock cannot be obtained, no locks are acquired
    * @internal
+   * @throws ConflictingLocksError
```

Comment:
```suggestion
   * @throws ConflictingLocksError if one or more requested locks are held by other briefcases.
```

I assume that's the condition under which this error is thrown.